test/setup.ts

```diff
@@ -151,7 +151,7 @@ declare global {
     type MockInstance<T, Y extends any[]> = JestMockInstance<(...args: Y) => T>;
 
     interface Mock<T = any, Y extends any[] = any>
-      extends Function, // eslint-disable-line @typescript-eslint/ban-types
+      extends Function,
```

Comment:
ESLint is complaining about this line. I'll let you figure out how to fix it. 😄

Reply:
Argh, the lint auto fixer removes the line every time. 

@viceice Do you know which rule removes this?